nodes/src/nodes/ml_sklearn/IGlobal.py

```
@@ -0,0 +1,17 @@
+
+class IGlobal:
+    def __init__(self, IEndpoint):
+        self.IEndpoint = IEndpoint
+        self.glb = None
+        self.preprocessor = None
+
+    def beginGlobal(self):
+        self.glb = self.IEndpoint.endpoint.glb
+        try:
+            self.preprocessor = PreProcessor()
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)